src/Misc/MiddleWare.cpp

```diff
@@ -1007,20 +973,22 @@ class MiddleWareImpl
     //Only valid until freed
     Master *previous_master = nullptr;
 
-    //The ONLY means that any chunk of UI code should have for interacting with the
-    //backend
-    Fl_Osc_Interface *osc;
     //Synth Engine Parameters
     ParamStore kits;
 
     //Callback When Waiting on async events
     void(*idle)(void*);
     void* idle_ptr;
 
-    //General UI callback
-    cb_t cb;
-    //UI handle
-    void *ui;
+    //General UI callbacks
+    cb_t cb[2];
+    //UI handles
+    void *ui[2];
+
+    //The ONLY means that any chunk of UI code should have for interacting with the
```

Comment:
Clarifying comment appreciated

Reply:
If you mean the "The ONLY means ..." comment, this one is from you in 6634e39ecf07190434709db24aa83ed8761357d0 , so I did not touch it. Whatever you find more explanatory, let me know and I will change it.

Reply:
heh, that's on me for not remembering my own handywork :p